JavaFXUserInterface/src/main/java/utilities/UIUtilities.java

```diff
@@ -153,10 +168,12 @@ public static boolean compareThresholds(TextField previousThreshold, TextField n
      */
     public void changeScene(String fxmlFileName, Scene scene) {
         try {
+            ((Controller) ((FXMLLoader) scene.getUserData()).getController()).getTimelines().forEach(Timeline::stop);
```

Comment:
yes, the one on ligne 189 calls another changescene method it does not do the scene change itself and thus does not need the timelines and controller as it would be redundant